SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -119,19 +74,18 @@ DrawGrid {
 DrawGridX {
 
 	var <grid,<>range,<>bounds;
-	var <>font,<>fontColor,<>gridColor,<>labelOffset;
+	var <font,<fontColor,<gridColor,<labelOffset;
```

Comment:
while you are at it, could you add spaces after the commas? Thanks!

Reply:
I actually rolled back a bunch of white space formatting changes in this PR, based on your [comment](https://github.com/supercollider/supercollider/pull/5161#issuecomment-1236568038) on in #5161. 

I was thinking of doing a separate PR that just overhauls all the style problems of this file at once. Sound OK?

Reply:
OK!